pkgs/applications/networking/browsers/microsoft-edge/browser.nix

```diff
@@ -126,10 +123,6 @@ stdenv.mkDerivation rec {
       --set-rpath "${libPath.libGLESv2}" \
       opt/microsoft/${shortName}/libGLESv2.so
 
-    patchelf \
-      --set-rpath "${libPath.libsmartscreen}" \
-      opt/microsoft/${shortName}/libsmartscreen.so
```

Comment:
I know you've removed this because it breaks dev but by removing it have you broken beta or main?

Reply:
All channels no longer include libsmartscreen.so. Microsoft has deprecated the old version of SmartScreen around version 108.

Reply:
In support of @JonaEnz's point: Note that `libsmartscreenn` (with the double `n` at the end), is apparently the *new* Microsoft Smartscreen (for example, see [source 1](https://techcommunity.microsoft.com/t5/discussions/smartscreen-is-cause-of-consistent-page-download-delays/m-p/3572750/highlight/true#M54146) and [source 2](https://endpointcave.com/how-to-configure-microsoft-defender-smartscreen-via-microsoft-intune/)).

Reply:
bump